packages/utils/src/document/reconversion-kickoff-loader.ts

```diff
@@ -0,0 +1,159 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { ReconversionKickoffParams } from "@metriport/core/command/reconversion/reconversion-kickoff-direct";
+import { SQSClient } from "@metriport/core/external/aws/sqs";
+import { getEnvVarOrFail } from "@metriport/core/util";
+import { uuidv7 } from "@metriport/core/util/uuid-v7";
+import { createUuidFromText } from "@metriport/shared/common/uuid";
+import { JSONParser, ParsedElementInfo } from "@streamparser/json";
+import fs from "fs";
+import { chunk, groupBy } from "lodash";
+
+/**
+ * This script is used to kick off a reconversion for a specific list of patients, and for a specific date range.
+ * It will send a message to the reconversion kickoff queue for each patient chunk (recommended to be 10/chunk).
+ *
+ * The reconversion kickoff lambda will then pick up the message and ping the API to reconvert patient documents.
+ *
+ * The input file is generated using the following SQL command:
+ * ```
+ * SELECT patient_id, cx_id
+ * FROM docref_mapping
+ * WHERE created_at BETWEEN <dateFrom> AND <dateTo>;
+ * ```
+ * Once the query is executed, export the results to a JSON file. Then, remove the SQL query from the object, and
+ * only leave the array of objects. Indicate the path to the file in the `fileName` variable.
+ *
+ * To run the script, execute:
+ * - ts-node src/document/reconversion-kickoff-loader.ts
+ */
+
+const sqsClient = new SQSClient({ region: getEnvVarOrFail("AWS_REGION") });
+const sqsUrl = getEnvVarOrFail("RECONVERSION_KICKOFF_QUEUE_URL");
+
+const PATIENT_CHUNK_SIZE = 10;
+
+/**
+ * AWS SQS limit is 3000 messages/second, but we'll be safe and use 2000
+ *
+ * @see https://aws.amazon.com/sqs/faqs/#topic-3:~:text=What%20is%20the%20throughput%20quota%20for%20an%20Amazon%20SQS%20FIFO%20queue%3F
+ */
+const SQS_BATCH_SIZE = 2000;
+
+const fileName =
+  "/Users/ramilgaripov/Desktop/metriport/full_stack/metriport/packages/utils/src/document/localReconversion.json";
+const dateFrom = "2025-07-10";
+const dateTo = "";
+
+/**
+ * Loads data from a large JSON file using streaming to avoid memory issues
+ */
+async function loadDataFromLargeJsonFile(
```

Comment:
nit: secondary functions go after the main one as defined in our standard and most up-to-date scripts